src/main/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/impl/TableMapping.java

```diff
@@ -241,21 +244,26 @@ void validateSecondaryIndexes(DynamoTableDescription virtualTable,
                 physicalSI = secondaryIndexMapper.lookupPhysicalSecondaryIndex(virtualSI, physicalTable);
             } catch (IllegalArgumentException | NullPointerException | MappingException e) {
                 throw new IllegalArgumentException("failure mapping virtual to physical " + virtualSI.getType() + ": " + e.getMessage() +
-                        ", virtualSIPrimaryKey=" + virtualSI + ", virtualTable=" + virtualTable + ", physicalTable=" + physicalTable);
+                    ", virtualSIPrimaryKey=" + virtualSI + ", virtualTable=" + virtualTable + ", physicalTable=" + physicalTable);
             }
             try {
                 // validate each virtual against the physical index that it was mapped to
                 validateCompatiblePrimaryKey(virtualSI.getPrimaryKey(), physicalSI.getPrimaryKey());
             } catch (IllegalArgumentException | NullPointerException e) {
                 throw new IllegalArgumentException("invalid mapping virtual to physical " + virtualSI.getType() + ": " + e.getMessage() +
-                        ", virtualSIPrimaryKey=" + virtualSI.getPrimaryKey() + ", physicalSIPrimaryKey=" + physicalSI.getPrimaryKey() +
-                        ", virtualTable=" + virtualTable + ", physicalTable=" + physicalTable);
+                    ", virtualSIPrimaryKey=" + virtualSI.getPrimaryKey() + ", physicalSIPrimaryKey=" + physicalSI.getPrimaryKey() +
+                    ", virtualTable=" + virtualTable + ", physicalTable=" + physicalTable);
             }
         }
 
         validateLSIMappings(virtualTable, physicalTable, secondaryIndexMapper);
     }
 
+    /*
```

Comment:
this seems like it actually may be a mistake...(?)

Reply:
Indeed.  I think the space between the javadoc and the method it was doc'ing must have threw off intellij.  I will fix.